api/src/main/java/org/openmrs/OrderGroup.java

```diff
@@ -232,6 +235,7 @@ public void setParentOrderGroup(OrderGroup parentOrderGroup) {
 	}
 
 	/**
+<<<<<<< HEAD
```

Comment:
Thanks for this @gitcliff Did you intend to leave it there when resolving merge conflicts

Reply:
oh sure ,,, let me remove it